core/build.gradle

```diff
@@ -0,0 +1,157 @@
+//TODO: check if this is really needed!!
+
+//buildscript {
```

Comment:
I don't link comments in source code 😅 is this really needed?

Reply:
Hmm... true enough, and I noticed that the build.gradle has a conflict, so I'll pick that up later today!